ephios/core/forms/events.py

```diff
@@ -207,3 +209,36 @@ def render(self):
 
     def save(self):
         pass
+
+
+class EventNotificationForm(forms.Form):
+    NEW_EVENT = "new"
+    REMINDER = "remind"
+    PARTICIPANTS = "participants"
```

Comment:
Elsewhere we are using `models.TextChoices`. Feel free to do that here as well.

Reply:
or IntegerChoice for that matter, just FYI

Reply:
well but we are are form and we don't have a model